pkg/cloudprovider/gce/gce.go

```diff
@@ -444,6 +490,19 @@ func (gce *GCECloud) EnsureTCPLoadBalancerDeleted(name, region string) error {
 			return err
 		}
 	}
+	fwName := makeFirewallName(name)
```

Comment:
Reversing would have meant deleting the firewall first, but don't worry about it